Broken tsconfig.app.json

### Demonstration
Create Angular parcel.
```
USER:spa$ npx create-single-spa --dir . --moduleType app-parcel --framework angular
? Project name (can use letters, numbers, dash or underscore) ng11error
? Do you want to enforce stricter type checking and stricter bundle budgets in the workspace?
  This setting helps improve maintainability and catch bugs ahead of time.
  For more information, see https://angular.io/strict No
? Would you like to add Angular routing? No
? Which stylesheet format would you like to use? CSS
CREATE ng11error/README.md (1018 bytes)
CREATE ng11error/.editorconfig (274 bytes)
CREATE ng11error/.gitignore (631 bytes)
CREATE ng11error/angular.json (3559 bytes)
CREATE ng11error/package.json (1199 bytes)
CREATE ng11error/tsconfig.json (458 bytes)
CREATE ng11error/tslint.json (3185 bytes)
CREATE ng11error/.browserslistrc (703 bytes)
CREATE ng11error/karma.conf.js (1104 bytes)
CREATE ng11error/tsconfig.app.json (287 bytes)
CREATE ng11error/tsconfig.spec.json (333 bytes)
CREATE ng11error/src/favicon.ico (948 bytes)
CREATE ng11error/src/index.html (295 bytes)
CREATE ng11error/src/main.ts (372 bytes)
CREATE ng11error/src/polyfills.ts (2826 bytes)
CREATE ng11error/src/styles.css (80 bytes)
CREATE ng11error/src/test.ts (753 bytes)
CREATE ng11error/src/assets/.gitkeep (0 bytes)
CREATE ng11error/src/environments/environment.prod.ts (51 bytes)
CREATE ng11error/src/environments/environment.ts (662 bytes)
CREATE ng11error/src/app/app.module.ts (314 bytes)
CREATE ng11error/src/app/app.component.css (0 bytes)
CREATE ng11error/src/app/app.component.html (25725 bytes)
CREATE ng11error/src/app/app.component.spec.ts (949 bytes)
CREATE ng11error/src/app/app.component.ts (213 bytes)
CREATE ng11error/e2e/protractor.conf.js (904 bytes)
CREATE ng11error/e2e/tsconfig.json (274 bytes)
CREATE ng11error/e2e/src/app.e2e-spec.ts (660 bytes)
CREATE ng11error/e2e/src/app.po.ts (274 bytes)
✔ Packages installed successfully.
    Successfully initialized git.
Installing packages for tooling via npm.
Installed packages for tooling via npm.
? Does your application use Angular routing? No
? Does your application use BrowserAnimationsModule? No
    Added 'single-spa' as a dependency
    Added 'single-spa-angular' as a dependency
    Added '@angular-builders/custom-webpack' as a dependency
    Generated 'main.single-spa.ts
    Generated 'single-spa-props.ts
    Generated asset-url.ts
    Generated extra-webpack.config.js
    Using @angular-devkit/custom-webpack builder.
    Updated angular.json configuration
    @angular-builders/custom-webpack:browser
CREATE extra-webpack.config.js (303 bytes)
CREATE src/main.single-spa.ts (788 bytes)
CREATE src/app/empty-route/empty-route.component.ts (143 bytes)
CREATE src/single-spa/asset-url.ts (502 bytes)
CREATE src/single-spa/single-spa-props.ts (333 bytes)
UPDATE package.json (1560 bytes)
UPDATE tsconfig.app.json (188 bytes)
UPDATE angular.json (3756 bytes)
For further routing setup, see https://single-spa.js.org/docs/ecosystem-angular#configure-routes
Project setup complete!
Steps to test your Angular single-spa application:
1. Run 'npm run serve:single-spa:ng11error'
2. Go to http://single-spa-playground.org/playground/instant-test?name=ng11error&url=%2F%2Flocalhost%3A4200%2Fmain.js&framework=angular to see it working!
```

Monitoring the file, just before `UPDATE tsconfig.app.json (188 bytes)` it looks like this:
```json
/* To learn more about this file see: https://angular.io/config/tsconfig. */
{
  "extends": "./tsconfig.json",
  "compilerOptions": {
    "outDir": "./out-tsc/app",
    "types": []
  },
  "files": [
    "src/main.ts",
    "src/polyfills.ts"
  ],
  "include": [
    "src/**/*.d.ts"
  ]
}
```

After the update it changed to single quotes:
```json

  extends: './tsconfig.json',
  compilerOptions: {
    outDir: './out-tsc/app',
    types: [],
  },
  files: [
    'src/main.single-spa.ts',
  ],
  include: [
    'src/**/*.d.ts',
  ],
}
```

Notice the file is already broken even before patch or install.

Patch webpack ( #306 ) with `npm install @angular-builders/custom-webpack@latest`.
Run `npm install` to install single-spa-angular.

Run `npm run serve:single-spa:ng11error` to have the error:
```
An unhandled exception occurred: tsconfig.app.json:2:3 - error TS1327: String literal with double quotes expected.

2   extends: './tsconfig.json',
    ~~~~~~~
```

Strangely I just created some parcels early in the morning and they did not end up with messed tsconfig.app.json file.


### Expected Behavior
The created project installs and serves.

### Actual Behavior
Breaks on serving.

--- a/schematics/ng-add/index.ts
+++ b/schematics/ng-add/index.ts
@@ -23,7 +23,7 @@ import {
 import { normalize, join } from 'path';
 // The JSON5 format supports comments and all Angular projects,
 // starting from version 10, contain comments in `tsconfig` files.
-import * as JSON5 from 'json5';
+import { parse } from 'json5';
 
 import { addScripts } from './add-scripts';
 import { Schema as NgAddOptions } from './schema';
@@ -167,7 +167,7 @@ function updateTSConfig(host: Tree, clientProject: WorkspaceProject): void {
     return;
   }
 
-  const tsConfig = JSON5.parse(buffer.toString());
+  const tsConfig = parse(buffer.toString());
 
   if (!Array.isArray(tsConfig.files)) {
     return;
@@ -176,7 +176,7 @@ function updateTSConfig(host: Tree, clientProject: WorkspaceProject): void {
   // The "files" property will only contain path to `main.single-spa.ts` file,
   // because we remove `polyfills` from Webpack `entry` property.
   tsConfig.files = ['src/main.single-spa.ts'];
-  host.overwrite(tsConfigPath, JSON5.stringify(tsConfig, null, 2));
+  host.overwrite(tsConfigPath, JSON.stringify(tsConfig, null, 2));
 }
 
 export function addNPMScripts(options: NgAddOptions): Rule {

